src/site/content/en/fast/lazy-loading-images/index.md

```diff
@@ -0,0 +1,302 @@
+---
+layout: post
+title: lazy-loading images
```

Comment:
Capitalize the title and double-check other occurrences of "lazy-loading", I've noticed a few more "find and replace" misses.

Reply:
I found one which I think was missed due to a hard return.